Makefile

```diff
@@ -111,6 +117,9 @@ BIN := build/bin
 OBJ := build/obj
 INC := build/include/sameboy
 LIBDIR := build/lib
+PKGCONF_DIR := $(LIBDIR)/pkgconfig
+LIBS := $(LIBDIR)/libsameboy.a $(LIBDIR)/libsameboy.so
```

Comment:
It would be better if both static and shared libraries could be enabled/disabled. Most distros may only want shared, some only want static.